data/json/items/tool/explosives.json

```diff
@@ -1615,5 +1615,62 @@
       "explosion": { "power": 69200, "shrapnel": { "casing_mass": 12000, "fragment_mass": 600 } }
     },
     "flags": [ "BOMB", "TRADER_AVOID", "DANGEROUS" ]
+  },
+  {
+    "id": "jug_bomb",
+    "type": "TOOL",
+    "category": "weapons",
+    "name": { "str": "jug bomb" },
+    "description": "A glass jug with gasoline spilled on the inside and a small pouch of oxidzier powder next to it, upon mixing these creat a incredibly shock explosives bomb, when activited you mix the two ingreidents its best to throw it as it might go off in your hands!",
+    "weight": "1133 g",
+    "volume": "750 ml",
+    "price": 6000,
+    "price_postapoc": 1000,
+    "to_hit": -3,
+    "bashing": 4,
+    "material": [ "glass" ],
+    "symbol": "|",
+    "color": "white",
+    "explode_in_fire": false,
+    "explosion": { "power": 938 },
+    "use_action": {
+      "need_wielding": true,
+      "target": "jug_bomb_act",
+      "msg": "You mix the ingridents.",
+      "target_charges": 20,
+      "active": true,
+      "need_fire": 0,
+      "menu_text": "Mix the ingridents",
+      "type": "transform"
+    },
+    "flags": [ "BOMB" ]
+  },
+  {
+    "id": "jug_bomb_act",
+    "type": "TOOL",
+    "category": "weapons",
+    "name": { "str": "jug bomb (mixed)", "str_pl": "jug bombs (mixed)" },
+    "description": "A mixed jug bomb, it may explode any moment now.",
+    "weight": "1133 g",
+    "volume": "750 ml",
+    "price": 0,
+    "price_postapoc": 0,
+    "to_hit": -3,
+    "bashing": 4,
+    "material": [ "glass" ],
+    "symbol": "|",
+    "color": "white",
+    "turns_per_charge": 1,
+    "explode_in_fire": true,
+    "explosion": { "power": 150 },
+    "use_action": {
+      "type": "explosion",
+      "sound_volume": 0,
+      "sound_msg": "rattle…",
+      "no_deactivate_msg": "You've already lit mixed %s, try throwing it instead.",
```

Comment:
```suggestion
      "no_deactivate_msg": "You've already mix %s, try throwing it instead.",
```

Reply:
That correction looks backwards. I think it should be "You've already lit %s, try throwing it instead."

Reply:
Inactive jug bomb explicitly states that it doesn't require fire to activate. So I don't understand why is there this "lit" part.

Reply:
OK, if the mixing is the activation, then it should be "You've already mixed %s, try throwing it instead."

Reply:
I updated my suggestion.